highlight.io/pages/_app.tsx

```diff
@@ -26,7 +26,8 @@ Router.events.on('routeChangeComplete', () => {
 	nProgress.done()
 })
 
-H.init('4d7k1xeo', {
+H.init('1xdn0e03', {
```

Comment:
Just double-checking that this is intentional. It looks like it would change prod.

Reply:
great catch, it wasn't intentional!